### PR TITLE
Test automation GFE

### PR DESCRIPTION
**Pull request by maxrichardjr**
Re-PRing all Test Automation Files

### CODE REVIEW

File: test/Test-Automation-GFE/.gitignore.bak
Comment (raphaelfretter) [inline test/Test-Automation-GFE/.gitignore.bak:1]: Remove this file. Shouldn't need to be in the repo.

File: test/Test-Automation-GFE/src/Execution/formsWorkflowPart2_BAK.java
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/Execution/formsWorkflowPart2_BAK.java:1]: Do we need backups for existing files on the repo here?

File: test/Test-Automation-GFE/src/Execution/formsWorkflowPart3_BAK.java
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/Execution/formsWorkflowPart3_BAK.java:1]: Here too

File: test/Test-Automation-GFE/src/Execution/formsWorkflow_BAK.java
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/Execution/formsWorkflow_BAK.java:1]: And here.

File: test/Test-Automation-GFE/src/PP-Demo-1_allTestsChrome.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="RegressionSuite-PreProd-Demo1">
+
+<!--    -->
```
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/PP-Demo-1_allTestsChrome.xml:5]: empty comment

File: test/Test-Automation-GFE/src/PP-Demo-2_allTestsChrome.xml [hunk 2]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="RegressionSuite-PreProd">
+
+<!--    -->
```
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/PP-Demo-2_allTestsChrome.xml:5]: empty comment

File: test/Test-Automation-GFE/src/PP_SelectTestsChrome.xml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="RegressionSuite-PRE-PROD">
+<!--    -->
```
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/PP_SelectTestsChrome.xml:4]: empty comment

File: test/Test-Automation-GFE/src/Prod_AllTests-Demo-3.xml [hunk 4]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="RegressionSuite-Pre-Prod">
+
+<!--    -->
```
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/Prod_AllTests-Demo-3.xml:5]: empty comment

File: test/Test-Automation-GFE/src/Prod_AllTests.xml [hunk 5]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="RegressionSuite-Prod-Demo1">
+
+<!--    -->
```
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/Prod_AllTests.xml:5]: empty comment

File: test/Test-Automation-GFE/src/Prod_Debugging.xml [hunk 6]
```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="RegressionSuite-PROD">
+
+<!--    -->
```
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/Prod_Debugging.xml:5]: empty comment... you get the idea.

File: test/Test-Automation-GFE/src/zArchive/GuessingGame.java
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/zArchive/GuessingGame.java:1]: I doubt this is needed.

File: test/Test-Automation-GFE/src/zArchive/arrays.java
Comment (raphaelfretter) [inline test/Test-Automation-GFE/src/zArchive/arrays.java:1]: Or this.

**Review by raphaelfretter — CHANGES_REQUESTED** [review]

File: test/Test-Automation-GFE/.gitignore.bak
Comment (shaneodd) [inline test/Test-Automation-GFE/.gitignore.bak:1]: Probably do not need the bak file anymore now that it is part of the main branch.

**Review by shaneodd — CHANGES_REQUESTED** [review]
Please see the comment on bak file, I see there are a couple other ones in there from the JAVA side of things as well. Now that we all can see and track these changes better.

**Review by Pelentan — APPROVED** [review]

**Pull request merged**